frontend/src/app/components/Header/tests.js

```diff
@@ -120,14 +120,6 @@ describe("Header", () => {
           .simulate("click", mockClickEvent);
       };
 
-      it("should ping GA and show retire dialog on retire item click", () => {
-        clickItem("menuRetire");
-        expect(preventDefault.called).to.be.true;
-        expect(subject.state("showRetireDialog")).to.be.true;
-        expect(subject.find("RetireConfirmationDialog")).to.have.property("length", 1);
-        expectMenuGA("Retire");
```

Comment:
this is already tested in RetireConfirmationDialog and the addition of react-router to that component broke this test